rimage/transform/cmd/extrinsic_calibration/verify_main_test.go

```diff
@@ -0,0 +1,12 @@
+package main
```

Comment:
Why was this file needed?

Reply:
I wasn't able to locally pass tests without adding it - I will admit I applied cargo-cult thinking ("all other packages have verify_main_test.go, this package needs it too)